test/integration/test-amp-ad-doubleclick.js

```diff
@@ -51,6 +51,7 @@ describes.sandboxed('Rendering of one ad', {}, () => {
         expect(iframe.src).to.contain('categoryExclusions');
         expect(iframe.src).to.contain('health');
         expect(iframe.src).to.contain('tagForChildDirectedTreatment');
+        expect(iframe.src).to.contain('maxAdContentRating');
```

Comment:
This test is currently being skipped, but if it weren't, I would expect it to fail? We do not expect `maxAdContentRating` to be in the ad request (iframe src), that would be `macr`, no?